CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ if(APPLE)
 	set_property(DIRECTORY
 		APPEND
 		PROPERTY COMPILE_DEFINITIONS SC_DARWIN)
+	set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/Install")
```

Comment:
> "...and the command line arguments passed to cmake: if a user sets the install prefix, you simply overwrite what the user requests."
Not from what I found. -DCMAKE_INSTALL_PREFIX at configure time still overrides this setting. There is no "force" involved.
> "fwiw, on osx there is no reason to use "install" anyway, as app bundles are self contained and osx isn't really fhs-compliant, anyway"
Many roads lead to Rome... I don't see a problem in adding a convenience for some that does not conflict with the workflow you propose.